scrapper.py

```diff
@@ -26,38 +39,140 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        articles = article_bs.find('div', {'class': 'entry-content'})
+        all_links = articles.find_all('a')[1:]
+        for link in all_links:
+            if len(self.urls) < self.max_articles:
+                if re.match(r'https?://vestnik\.lunn\.ru/', link['href']):
```

Comment:
? we already have a domain name. to this moment using regex seems to be overkill, or improve regex